### ISSUE TITLE
More website/software icons

### ISSUE THREAD

**Issue by Efreak** [winauth/winauth#299]
I've got some icons here. The images below are all 48x48 as used by WinAuth; the links will take you to the original images I used.

[![digital ocean](http://a.pomf.cat/dninwj.png)](https://upload.wikimedia.org/wikipedia/en/f/ff/DigitalOcean_logo.svg) [![Ebay](http://a.pomf.cat/puoupj.png)](http://a.pomf.cat/ehwvwa.png) [![itch.io](http://a.pomf.cat/ldocpi.png)](http://a.pomf.cat/ztqmzi.png) [![name.com](http://a.pomf.cat/ugqyon.png)](http://a.pomf.cat/gldfba.png) [![origin](http://a.pomf.cat/lduyvs.png)](http://a.pomf.cat/mxevwv.jpg) [![PayPal](http://a.pomf.cat/gsktdu.png)](http://a.pomf.cat/ixqfwr.png) [![Dreamhost](http://a.pomf.cat/dnjwjp.png)](http://a.pomf.cat/cvhrui.png) [![tumblr](http://a.pomf.cat/hbkstb.png)](http://a.pomf.cat/lsusbz.png) [![tumblr-official](http://a.pomf.cat/bcuwfp.png)](http://a.pomf.cat/kzqnwf.png) [![tumblr-color](http://a.pomf.cat/nejmqt.png)](https://gist.githubusercontent.com/Efreak/ace66d0028724c0f12eb/raw/775795c95fe96e365937307bd5cbfae88775b0c9/tumblr-color.svg) [![google](http://a.pomf.cat/dtahtd.png)](https://gist.github.com/Efreak/f8506771a221e688df6b) [![gmail](http://a.pomf.cat/xnmavh.png)](https://upload.wikimedia.org/wikipedia/commons/4/45/New_Logo_Gmail.svg) [![dropbox-outline](http://a.pomf.cat/hgubgo.png) ![dropbox-blue](http://a.pomf.cat/acaqqg.png)](https://www.dropbox.com/branding) [![gitlab](http://a.pomf.cat/trxejg.png)](https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/logo/logo.svg) [![gitlab-fox](http://a.pomf.cat/mzssrm.png)](https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/archive/cake/fox.svg) [![gitlab](http://a.pomf.cat/pazsvz.png)](http://a.pomf.cat/oyoqud.png) ![gitlab](http://a.pomf.cat/eccmnn.png) ![gitlab](http://a.pomf.cat/sjneui.png) [![gitlab](http://a.pomf.cat/yedtkl.png)](http://a.pomf.cat/uzyqrb.jpg) [![bitbucket](http://a.pomf.cat/sxjtvs.png)](https://design.atlassian.com/how-we-design/resources/) [![yahoo](http://a.pomf.cat/vjlctn.png)](https://www.yahoo.com/sy/os/mit/media/p/common/images/favicon_new-7483e38.svg) [![GitHub](http://a.pomf.cat/gibmjx.png) ![GitHub](http://a.pomf.cat/suygfm.png)](https://github.com/logos) [![EA](http://a.pomf.cat/mgzwfh.png)](http://vignette1.wikia.nocookie.net/u5lazarus/images/e/eb/EA-Logo.svg/revision/latest) [![EA](http://a.pomf.cat/kwaoon.png) ![EA](http://a.pomf.cat/qowfzq.png)](https://upload.wikimedia.org/wikipedia/commons/0/0d/Electronic-Arts-Logo.svg) [![Chrome](http://a.pomf.cat/hokerj.png)](https://upload.wikimedia.org/wikipedia/commons/0/01/Google_Chrome_icon_and_wordmark_(2011).svg) [![Wordpress](http://a.pomf.cat/fpoibn.png) ![Wordpress blue](http://a.pomf.cat/ulauge.png)](https://upload.wikimedia.org/wikipedia/commons/2/20/WordPress_logo.svg) [![TeamViewer](http://a.pomf.cat/awhecf.png)](http://www.teamviewer.com/resources/images/logos/teamviewer-logo-big.svg) [![LastPass](http://a.pomf.cat/rapzue.png)](https://lastpass.com/media/pressroom/LastPass-Logo-Color.svg) [![amazon](http://a.pomf.cat/vphqvr.png)](https://pbs.twimg.com/media/CU0xEY9UsAEu0B4.jpg) [![IFTTT](http://a.pomf.cat/cjuevy.png)](https://d3rnbxvnd0hlox.cloudfront.net/assets/r2012/ifttt_logo_83-2fa98a5d840c5fa967eb43431c900da7.svg) [![IFTTT](http://a.pomf.cat/wpedxl.png) ![IFTTT](http://a.pomf.cat/wihcih.png)](http://marketingland.com/wp-content/ml-loads/2012/09/ifttt-logo.jpg) [![KickStarter](http://a.pomf.cat/zxswrx.png)](https://worldvectorlogo.com/logos/kickstarter-2.svg)

I should note that paypal and ebay use Verisign VIP Access; you can't just get the shared secret from them directly. You may want to leave them out for that reason. The colorful tumblr icon is their favicon; I also included the current official icon (the shadow/3d look) and the past official icon (the flat one with round corners).

**Comment by linhongjun** [comment]
Do you have icon for dropbox?

**Comment by winauth** [comment]
Thanks. The icon chooser needs to be better now :)

**Comment by thorerik** [comment]
Gitlab would also be appriciated

**Comment by Efreak** [comment]
Added more icons. to the post. Again, inline images are 48x48, but this time the links are the original sources (except the google one, which I made myself).

There's a bunch of choices for gitlab; the orange one seems to be their primary logo, but maybe someone wants the fox?

The GitHub logo is outdated (or incorrect; I have no idea which) so I grabbed that one as well, even though it's already in there.

The chrome icon is slightly larger, so it should fill the box available.

I also redid the digital ocean logo from the svg on wikipedia; the coloring is somewhat lighter on this one, and I think it looks nicer.
